app/controllers/admin/collections_controller.rb

```diff
@@ -118,6 +120,18 @@ def set_collection
       @collection = Collection.find_by_friendlier_id!(params[:id])
     end
 
+    # Params method just for #index.
+    # Also sets default sort column and order.
+    # This does not actually perform any sorting or filtering.
+    def index_params
+      @index_params ||= Kithe::Parameters.new(params).permit(
```

Comment:
You don't need to use Kithe::Parameters here, let's just use ordinary rails strong params. https://api.rubyonrails.org/classes/ActionController/StrongParameters.html

Which means this can just be `params.permit...`